prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -361,6 +377,81 @@ ssize_t rxr_pkt_post_ctrl_or_queue(struct rxr_ep *ep, int entry_type, void *x_en
 	return err;
 }
 
+/*
+ * This function should be called before sending a message with the new protocol
+ * for FI_DELIVERY_COMPLETE.
+ *
+ * ep: The endpoint on which the packet for triggering handshake will be sent.
+ * peer: The peer from which the sender receives handshake.
+ * source_tx_entry: The tx_entry for the trigger packet.
+ *
```

Comment:
Please add a brief introduction about what this function will do: it will send an eager rtw packet with no data, then keep calling ```rxr_ep_progress_internal``` util rxr_peer-flags has the HANDSHAKE_RECEIVED flag. Upon receiving the eager RTW, the receiver will post a handshake, which will be processed in  sender's ```rxr_ep_progress_internal```